WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -162,9 +162,13 @@ static IEnumerable<(int Size, FeeRate From, FeeRate To)> SplitFeeGroupInBlocks(F
 		// Filter those groups with very high fee transactions (less than 0.1%).
 		// This is because in case a few transactions pay unreasonably high fees
 		// then we don't want our estimations to be affected by those rare cases.
-		var relevantFeeGroups = mempoolInfo.Histogram
-			.OrderByDescending(x => x.Group)
-			.SkipWhile(x => x.Count < mempoolInfo.Size / 1_000);
+		var histogram = mempoolInfo.Histogram.OrderByDescending(x => x.Group).ToArray();
```

Comment:
Does the comment above this peace of code need to be updated?

Reply:
no.